gitreceive/receiver/flynn-receive.go

```diff
@@ -189,23 +199,25 @@ Options:
 			Processes: map[string]int{"web": 1},
 		}
 
-		watcher, err := client.WatchJobEvents(app.ID, release.ID)
-		if err != nil {
-			log.Fatalln("Error streaming job events", err)
-			return
-		}
-		defer watcher.Close()
```

Comment:
Yes, I am superseding his authority on this one :wink:

The deployment waits for job events and emits deployment events, so we should just be logging those deployment events and waiting for either a `failed` or `completed` deployment event (which will also fix #2159).